blocks/api/serializer.js

```diff
@@ -37,8 +39,18 @@ export function getSaveContent( save, attributes ) {
 		}
 	}
 
+	// Adding a generic classname
+	const addClassnameToElement = ( element ) => {
```

Comment:
It'd be nice to have a list of blocks where we don't want to apply the class (`p` and `heading`, to start).